command/core.py

```diff
@@ -19,17 +20,19 @@ def handle_app_mention(self, event_data):
         message = event_data["event"]["text"]
         user = event_data["event"]["user"]
         channel = event_data["event"]["channel"]
-        s = message.split(' ', 2)
-        if s[0] != "@rocket":
+
+        if not message.startswith('<@U') and not message.startswith('@'):
```

Comment:
The "right way" to solve this is probably to lookup the encoded ID for Rocket on startup (which should be added to our Slack interface) and then checking for that. Probably not necessary to do that for this PR, but good to keep in mind.